invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -293,11 +296,18 @@ def dashboard(dashboard_name=None):
     """Display user dashboard page."""
     if not current_app.config["COMMUNITIES_ENABLED"] or not dashboard_name:
         dashboard_name = current_app.config["_DASHBOARD_ROUTES"][0]
+    prefix = current_user_resources.users_resource.config.url_prefix
+    route = current_user_resources.users_resource.config.routes["item-avatar"]
+    image = '/api{prefix}{route}'.format(
+        prefix=prefix,
+        route=re.sub("<.+>", str(current_user.id), route)
+    )
     return render_template(
         "invenio_app_rdm/records/dashboard.html",
         dashboard_name=dashboard_name,
         searchbar_config=dict(searchUrl=get_search_url()),
-        username=current_userprofile.full_name
+        username=current_userprofile.full_name,
+        image=image
```

Comment:
```suggestion
        current_user_avatar=image
```
for better readability

Reply:
I did this but push didn't go through my bad.